HotelBooking/Views/book_reservation_view.py

```diff
@@ -130,11 +131,27 @@ def reserve_room(self):
             if self.user_id == None:
                 self.user_id = self.prompt_and_get_answer(
                     PROMPT_KEY['CUSTOMER'])
-            self.start_date = self.prompt_and_get_answer(
-                PROMPT_KEY['START_DATE'])
-            self.duration = int(
-                self.prompt_and_get_answer(PROMPT_KEY['DURATION']))
 
+            while(True):
```

Comment:
Don't do this, use the validate field of PyInquire, i.e.:
```
            {
                'type': 'input',
                'message': "Enter transaction amount in cents",
                'name': 'amount',
                'validate': lambda x: x.isdigit() or "Transaction amount invalid",
            },
```

Reply:
Ill check it out